apps/web-giddh/src/app/settings/personal-information/personal-information.component.ts

```diff
@@ -58,27 +58,98 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
     public isCopied: boolean = false;
     /** This will hold portal url */
     public portalUrl: any = PORTAL_URL;
-    /** Holds Last Portal Domain */
-    private tempProfileData: any = null;
+    /** Holds Profile Form */
+    public profileForm: FormGroup;
 
-    constructor(private generalService: GeneralService, private toasty: ToasterService, private clipboardService: ClipboardService) { }
+    constructor(private generalService: GeneralService, private toasty: ToasterService, private clipboardService: ClipboardService, private formBuilder: FormBuilder) { }
 
     /**
      * Initializes the component
      *
      * @memberof PersonalInformationComponent
      */
     public ngOnInit(): void {
-        this.tempProfileData = this.profileData;
+        this.initProfileForm();
         this.voucherApiVersion = this.generalService.voucherApiVersion;
         this.isValidDomain = this.generalService.checkDashCharacterNumberPattern(this.profileData.portalDomain);
         this.saveProfileSubject.pipe(takeUntil(this.destroyed$)).subscribe((res) => {
-            if (res) {
+            if (res && this.profileForm.dirty) {
```

Comment:
Check the condition

Reply:
Checked

Reply:
confirm with @ravindersinghdang sir for this condition